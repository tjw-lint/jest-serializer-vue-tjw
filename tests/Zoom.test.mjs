const helpers = require('./helpers.js');

import { mount } from '@vue/test-utils';
import Zoom from './components/Zoom.vue';

describe('Zoom.vue', () => {
  let wrapper;

  beforeEach(() => {
    helpers.mockSettings({});
    wrapper = mount(Zoom);
  });

  test('Zoom in on a child element', () => {
    const article = wrapper.find('[data-test="article"]');

    expect(article)
      .toMatchSnapshot();
  });

  test('Zoom in on a table row', () => {
    const tr = wrapper.find('[data-test="tr"]');

    expect(tr)
      .toMatchSnapshot();
  });
});
