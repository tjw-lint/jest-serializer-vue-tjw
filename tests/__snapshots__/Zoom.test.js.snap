// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Zoom.vue Zoom in on a child element 1`] = `
<article>
  <p>
    Lorem ipsum dolor sit amet, <strong>consectetur adipisicing</strong> elit.
  </p>
</article>
`;

exports[`Zoom.vue Zoom in on a table row 1`] = `
<tr>
  <td>
    first
  </td>
  <td>
    second
  </td>
</tr>
`;
